packages/x-components/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+const { xTailwindPlugin } = require('@empathyco/x-tailwindcss');
+
+module.exports = {
+  content: [
+    './index.html',
+    './src/components/**/*.vue',
+    './src/views/**/*.vue',
```

Comment:
Could we simply include all `.vue` files that are under `src`?: `./src/**/*.vue`